OutOfSchool/OutOfSchool.BusinessLogic/Util/Mapping/WorkshopDraftMappingProfile.cs

```diff
@@ -33,26 +33,86 @@ public WorkshopDraftMappingProfile()
+      
+        CreateMap<WorkshopV2Dto, WorkshopDraftResponseDto>()
+            .ForMember(dest => dest.WorkshopDraftId, opt => opt.Ignore())
+            .ForMember(dest => dest.WorkshopDetails, opt => opt.MapFrom(src => src))
+            .ForMember(dest => dest.DraftStatus, opt => opt.Ignore())
+            .ForMember(dest => dest.RejectionMessage, opt => opt.Ignore())
+            .ForMember(dest => dest.OwnershipType, opt => opt.Ignore())
+            .ForMember(dest => dest.IncludedStudyGroupsIds, opt => opt.Ignore())
+            .ForMember(dest => dest.WorkshopStatus, opt => opt.Ignore())
+            
+            .ForMember(dest => dest.DateTimeRanges, opt => opt.Ignore())
+            .ForMember(dest => dest.WorkshopDescriptionItems, opt => opt.Ignore());
+        
+        CreateMap<Workshop, WorkshopDraftViewCardDto>()
+           .ForMember(dest => dest.WorkshopDraftId, opt => opt.MapFrom(src => src.Id)) // Assuming `WorkshopDraftId` corresponds to `Workshop.Id`
+           .ForMember(dest => dest.Title, opt => opt.MapFrom(src => src.Title))
+           .ForMember(dest => dest.DraftStatus, opt => opt.Ignore())
+           .ForMember(dest => dest.RejectionMessage, opt => opt.Ignore());
+
+        CreateMap<Workshop, WorkshopDraft>()
+            .ForMember(dest => dest.DraftStatus, opt => opt.Ignore())
+            .ForMember(dest => dest.Id, opt => opt.Ignore())
+            .ForMember(dest => dest.Teachers, opt => opt.Ignore())
+            .ForMember(dest => dest.Images, opt => opt.Ignore())
+            .ForMember(dest => dest.RejectionMessage, opt => opt.Ignore())
+            .ForMember(dest => dest.ModifiedAt, opt => opt.Ignore())            
+            .ForMember(dest => dest.Version, opt => opt.Ignore())
+            .ForMember(dest => dest.WorkshopDraftContent, opt => opt.Ignore())
+            .ForMember(dest => dest.Workshop, opt => opt.Ignore())            
+            .ForMember(dest => dest.CreatedAt, opt => opt.Ignore())            
+            .ForMember(dest => dest.ModifiedBy, opt => opt.MapFrom(src => src.ModifiedBy))            
+            .ForMember(dest => dest.CoverImageId, opt => opt.MapFrom(src => src.CoverImageId))
+            .ForMember(dest => dest.ProviderId, opt => opt.MapFrom(src => src.ProviderId))
+            .ForMember(dest => dest.WorkshopId, opt => opt.MapFrom(src => src.Id));            
+
+
+        CreateMap<WorkshopDraft, WorkshopDraftViewCardDto>()
+            .ForMember(dest => dest.WorkshopDraftId, opt => opt.MapFrom(src => src.Id))
+            .ForMember(dest => dest.DraftStatus, opt => opt.MapFrom(src => src.DraftStatus))
+            .ForMember(dest => dest.Title, opt => opt.MapFrom(src => src.WorkshopDraftContent.Title))
+            .ForMember(dest => dest.RejectionMessage, opt => opt.MapFrom(src => src.RejectionMessage));            
+
+        CreateMap<WorkshopDraft, WorkshopDraftRejectionDto>()            
```

Comment:
`WorkshopDraftRejectionDto` is an input dto that comes into our system from API and i don't see it is being returned anywhere. Do we really need this mapping?